webui/react/src/components/TaskActionDropdown.tsx

```diff
@@ -51,23 +51,12 @@ const TaskActionDropdown: React.FC<Props> = ({ task }: Props) => {
     params.domEvent.stopPropagation();
     try {
       switch (params.key) { // Cases should match menu items.
-        case 'kill':
```

Comment:
suggestion: avoid moving the switch cases around to preserve history if there is no real change happening.

Reply:
I assume the move is to keep the order the same as the menu items?